.github/workflows/wpt.yml

```diff
@@ -18,9 +18,16 @@ jobs:
         with:
           go-version: 1.23.x
           check-latest: true
-      - name: Run tests
+      # TODO: combine WebPlatform tests checkout & patch into the singe step
+      - name: Run Streams Tests
         run: |
           set -x
           cd js/modules/k6/experimental/streams/tests
           sh checkout.sh
           go test ../... -tags=wpt
+      - name: Run Webcrypto Tests
+        run: |
+          set -x
+          cd js/modules/k6/webcrypto/tests
+          sh checkout.sh
+          go test ../... -tags=wpt
```

Comment:
Why does this run the tests starting from the parent directory?

Reply:
done

Reply:
done